inst/pages/98_exercises.qmd

```diff
@@ -651,6 +792,85 @@ Useful functions: nrow, ncol, dim, summary, table, quantile, unique, transformAs
    As for assays, you can access the desired altExp by name or index.  
 6. **Extra**: Split the data based on other features with `splitOn`.  
 
+### Beta Diversity using Alternative experiments
+
+This Exercise will focus on calcuating dissimilarities or beta diversity using alternative experiments.
+
+1. Import the mia and vegan packages and load a dataset. This can be your own or one of the packages built in to mia.
+
+```{r}
+#| label: beta-AltExp-ex1
+#| eval: FALSE
+#| code-fold: true
+#| code-summary: "Show solution"
+
+library(mia)
+library(vegan)
+
+data("Tengeler2020", package = "mia")
+tse <- Tengeler2020
+```
+
+2. Calculate relative abundances and create alternative experiments using `splitByRanks` or `splitOn`
```

Comment:
It might be better that the main task is not to split the data and put it to altExp() but rather agglomerate the data. That would be more meaningful

So, this could be "Agglomerate data to all available taxonomy ranks by using agglomerateByRanks()."

Also, the data transformation should be done after the agglomeration. Here the order does affect only to scale compared to if summing is done to counts (the maximum value is not 1 but still the relativeness holds). But there would problem with negative transformation for instance.

I think we should add option for applying transform also for altExps() in tranformAssay

Reply:
I agree that it would be good to extend to altExps in transformAssay. I thought we already had an issue on this but can't find. Could be opened. @Insaynoah could you open issue on that?

Reply:
We have actually issue on that https://github.com/microbiome/mia/issues/429 It is very old

Reply:
Is this resolved @Insaynoah